test/test_language_scanner.py

```diff
@@ -11,6 +11,8 @@
 from cve_bin_tool.util import ProductInfo
 from cve_bin_tool.version_scanner import VersionScanner
 
+from test.utils import LONG_TESTS
```

Comment:
You're failing a linter check because isort wants this line up directly below the "from pathlib" line.

If you've got isort installed, you can run it and it'll fix it so you can update the pull request:

```console
isort test/test_language_scanner.py
git add test/test_language_scanner.py
git commit -m "chore: fix isort error"
```

Then don't forget to push your branch  to github.